custom_components/elegoo_printer/image.py

```
@@ -0,0 +1,143 @@
+if TYPE_CHECKING:
+    import aiohttp
+    from homeassistant.config_entries import ConfigEntry
+    from homeassistant.core import HomeAssistant
+    from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+    from custom_components.elegoo_printer.coordinator import ElegooDataUpdateCoordinator
+
+COVER_IMAGE_SENSOR = ElegooPrinterSensorEntityDescription(
+    key="cover_image",
+    translation_key="cover_image",
+    value_fn=lambda self: "",
```

Ruff (0.8.2)

Unused lambda argument: `self`

(ARG005)